client/coral-framework/utils/user.js

```diff
@@ -35,6 +37,41 @@ export const isBanned = user => {
   return get(user, 'state.status.banned.status');
 };
 
+/**
+ * isUsernameRejected
+ * retrieves boolean based on the username status
+ */
+
+export const isUsernameRejected = user => {
+  return get(user, 'state.status.username.status') === 'REJECTED';
+};
+
+/**
+ * isUsernameChanged
+ * retrieves boolean based on the username status
+ */
+
+export const isUsernameChanged = user => {
+  return get(user, 'state.status.username.status') === 'CHANGED';
+};
+
+/**
+ * getActiveStatuses
+ * returns an array of active status(es)
+ * i.e if suspension is active, it returns suspension
+ */
+
+export const getActiveStatuses = user => {
+  const statusMap = {
+    suspended: isSuspended,
+    banned: isBanned,
+    usernameRejected: isUsernameRejected,
+    usernameChanged: isUsernameChanged,
+  };
+
+  return toPairs(mapValues(statusMap, fn => fn(user))).filter(x => x[1]);
```

Comment:
Should return a flat array instead of pairs. The pair has no additional information compared to a flat array and as such has no function.